CONTRIBUTING.md

```diff
@@ -1,27 +1,88 @@
-# Contributing to bids-starters-kit
+# Contributing to the BIDS Starter Kit
 
-Welcome to the BIDS Starters Kit repository! We're excited you're here and want to contribute.  
+:tada::balloon::cake: **Welcome to the BIDS Starters Kit repository!** :cake::balloon::tada:
 
-These guidelines are designed to make it as easy as possible to get involved. If you have any questions that aren't discussed below, please let us know by opening an [issue](https://github.com/INCF/bids-starter-kit/issues)!
+:dizzy::hatched_chick::sunny: *We're so excited you're here and want to contribute.* :sunny::hatched_chick::dizzy:
+
+The point of this starter kit is to **welcome new users and contributors to the BIDS community**. We hope that these guidelines are designed to make it as easy as possible to get involved. If you have any questions that aren't discussed below, please let us know through one of the many ways to [get in touch](#get-in-touch).
+
+## Table of contents
+
+* [Joining the BIDS community](#joining-the-community)
+* [Get in touch](#get-in-touch)
+* [Contributing through GitHub](#contributing-through-github)
+* [Where to start: wiki, code and templates](#where-to-start-wiki-code-and-templates)
+* [Where to start: labels](#where-to-start-labels)
+* [Make a change with a pull request](#making-a-change-with-a-pull-request)
+* [Recognizing contributions](#recognizing-contributions)
 
-Before you start you'll need to set up a free [GitHub](https://github.com/) account and sign in. Here are some [instructions](https://help.github.com/articles/signing-up-for-a-new-github-account/).
 
 ## Joining the community
 
-BIDS is a growing community of neuroimaging enthusiasts, and we want to make our resources accessible to and engaging for as many researchers as possible.
-We therefore require that all contributions adhere to our [Code of Conduct](http://www.brainhack.org/code-of-conduct.html).
+BIDS - the [Brain Imaging Data Structure][bids] - is a growing community of neuroimaging enthusiasts, and we want to make our resources accessible to and engaging for as many researchers as possible.
+
+We therefore require that all contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**.
+
+How do you know that you're a member of the BIDS community? You're here! You know that BIDS exists! You're officially a member of the community. It's THAT easy! Welcome! :joy::raised_hands:
+
+
+## Get in touch
+
+There are lots of ways to get in touch with the team maintaining the BIDS Starter Kit.
+
+* Our channel in the [BrainHack slack team][brainhack-slack-starterkit]
```

Comment:
So Kirstie, Chris G and I had a chat about this during the OHBM Hackathon. Like you said, we agreed that it may not ultimately be feasible to maintain every form of communication but for the time being:
a) I can reply
b) If we reach an amount of users where it becomes too much for the admin team to reply, the channels could become self-sustaining (ideally)
So we just decided to leave all channels open for now

Reply:
If there's commitment to monitor the channel, then I think the moderator(s) can take care of cross-linking concerns (I count myself in this list, of course !). And yes, Neurostars is definitely another forum we should cover.. do you think we should have our own tag, or is `BIDS` inclusive enough ?

Reply:
Okay I added the link to Neurostars with a short explanation and some example threads. 
Having a BIDS-starter-kit tag could be cool! But I think for now the amount of traffic on Neurostars is small enough that the BIDS umbrella tag works well enough.
And yea I agree it would be really important for us to have good cross-linking 👍